managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -153,7 +153,7 @@ To avoid any statement timeout errors with Managed Database for PostgreSQL (e.g.
     - **Password**
 
     <Message type="note">
-    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with.
```

Comment:
I forgot this one sorry

We also need to replace it with this: "You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. These are the username and password that you defined upon Database Instance creation."

Reply:
```suggestion
    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. These are the username and password that you defined upon Database Instance creation.
```